spec/numberformat.html

```diff
@@ -636,8 +638,7 @@ <h1>ToRawPrecision ( _x_, _minPrecision_, _maxPrecision_ )</h1>
           1. Let _e_ be 0.
           1. Let _xFinal_ be 0.
         1. Else,
```

Comment:
This algorithm really needs a `Set _x_ to ℝ(_x_).` step before perform any arithmetic or comparisons involving _x_, matching ECMA-262 [Number.prototype.toPrecision](https://tc39.es/ecma262/#sec-number.prototype.toprecision).